univocity-trader-cli/src/main/java/com/univocity/trader/cli/Main.java

```diff
@@ -1,5 +1,9 @@
 package com.univocity.trader.cli;
 
+import java.util.ArrayList;
```

Comment:
Can you please update your IDE code formatter to use star imports and use the tab delimiter for indentation?